crates/database/src/orders.rs

```diff
@@ -1809,10 +1850,11 @@ mod tests {
         .await
         .unwrap();
 
-        let orders: Vec<_> = order_parameters_with_most_outdated_fees(&mut db, timestamp, 2, 100)
-            .try_collect()
-            .await
-            .unwrap();
+        let orders: Vec<_> =
+            order_parameters_with_most_outdated_fees(&mut db, timestamp, 2, 100, true)
```

Comment:
This is why specifying parameters as bare `bool`s is not the best idea: it is not clear to the reader what this `true` stands for at a glance.